Song_sort_Genre.php

```diff
@@ -27,13 +27,13 @@
 	<div class ="grid-container">
 		<!-- The div for nav and in the div it contain the logo and a function to called the nav -->
 		<div class="navigation">
-			<!-- Set a div that contain logo for the music database -->
 			<div class="logo">
+				<!-- Set a div that contain logo for the music database -->
 				<a href = "#"><img src = "images/logo8.png" height = "100"></a>
 			</div>
 			<?php
-				//Pulls the links from the nav_normal.php page and places them in the navigation div
-				require 'nav_normal.php'; //'require' is 100% needed for the site to run 
+				//Pulls the links from the nav_admin.php page and places them in the navigation div
```

Comment:
21/9: I change to link of the nav normal to nav admin because nav admin and allowed to access to other function which was delete user for example.